src/lib/ndpi_utils.c

```diff
@@ -2199,6 +2230,11 @@ ndpi_http_method ndpi_http_str2method(const char* method, u_int16_t method_len)
       }
     }
     break;
+
+  case 'U':
+    if (method_len == 6)
+      return(NDPI_HTTP_METHOD_UNLOCK);
+    break;
   }
```

Comment:
Well, it works, but doesn't seem optimal to me. @IvanNardi what do you think? Is there any way this could be improved?